chamber/models/__init__.py

```diff
@@ -30,17 +29,22 @@ class Options(object):
     def __init__(self, model):
         self.model = model
 
-        self.clean_before_save = True
-        self.clean_after_save = False
+        self.clean_pre_save = True
```

Comment:
i dont like a fact that clean_pre_save is flag and _clean_pre_save is function. Can't you rename clean_pre_save to something like is_clean_pre_save or is_clean_pre_save_enabled?